packages/webcomponentsjs/ts_src/platform/toggle-attribute.ts

```diff
@@ -0,0 +1,41 @@
+/**
+@license
+Copyright (c) 2020 The Polymer Project Authors. All rights reserved.
+This code may only be used under the BSD style license found at
+http://polymer.github.io/LICENSE.txt The complete set of authors may be found at
+http://polymer.github.io/AUTHORS.txt The complete set of contributors may be
+found at http://polymer.github.io/CONTRIBUTORS.txt Code distributed by Google as
+part of the polymer project is also subject to an additional IP rights grant
+found at http://polymer.github.io/PATENTS.txt
+*/
+
+export {};
+
+const Element_prototype = Element.prototype;
```

Comment:
```suggestion
const Element_prototype = Element.prototype;
const nativeHasAttribute = Element.prototype.hasAttribute;
const nativeSetAttribute = Element.prototype.setAttribute;
const nativeRemoveAttribute = Element.prototype.removeAttribute;
```

These need to be saved to avoid calling back into other polyfills that were applied after this one.